index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en-us">
+
+<!--Head section of HTML-->
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title> Horiseon </title>
```

Comment:
The title could be more descriptive. Think about adding something that describes the company or their work e.g. SEO Specialists.